samples/v1/language_classify_gcs.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "language_classify_gcs")
+
+// sample-metadata:
+//   title: Classify Content (GCS)
+//   description: Classifying Content in text file stored in Cloud Storage
+//   usage: node samples/v1/language_classify_gcs.js [--gcs_content_uri "gs://cloud-samples-data/language/classify-entertainment.txt"]
+
+'use strict';
+
+// [START language_classify_gcs]
+
+const {LanguageServiceClient} = require('@google-cloud/language').v1;
+
+/**
+ * Classifying Content in text file stored in Cloud Storage
+ *
+ * @param gcsContentUri {string} Google Cloud Storage URI where the file content is located.
+ * e.g. gs://[Your Bucket]/[Path to File]
+ * The text file must include at least 20 words.
+ */
+function sampleClassifyText(gcsContentUri) {
```

Comment:
This is a bit nuanced, but we want to have an outer function named `main`, with the contents of the sample in side it.  The require call would be inside this function, and the region tags.  Pay special attention to the way the functions are wrapped here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

I know this is nitpicky, but this style of sample has _really_ helped reduce confuse for embedded snippets.

Reply:
Not nitpicky at all IMO!

We're rendering characters & they're all equally important, essentially 😎

Noted and added to the list!